### ISSUE TITLE
4.0.2 on RubyGems.org has different dependencies to the git repo

### ISSUE THREAD

**Issue by smcgivern** [middleman/middleman-blog#369]
In this repo, at the v4.0.2 tag: https://github.com/middleman/middleman-blog/blob/v4.0.2/middleman-blog.gemspec#L21

```
  s.add_dependency( "middleman-core", [ ">= 4.0.0" ] )
```

But on RubyGems.org, I see that this depends on `~> 4.0`: https://rubygems.org/gems/middleman-blog/versions/4.0.1

That means it's not possible to use the master branch of `middleman` with `middleman-blog` at 4.0.2, even though it seems like it should be allowed? Either way, the divergence between RubyGems.org and the project tag is confusing.

**Comment by tdreyno** [comment]
I agree, that is very odd.  I can't find anywhere in the git history where it was `~>`.

I'm going to double check that `master` is passing the build, then release a 4.0.3

**Comment by tdreyno** [comment]
Done: https://rubygems.org/gems/middleman-blog/versions/4.0.3
